microsoftgraph/microsoft-graph-explorer-v4
# Task: Fix auto-collapsing of permissions items

## Overview
Closes #2034 

### Demo

Optional. Screenshots, `curl` examples, etc.

### Notes

Optional. Ancillary topics, caveats, alternative strategies that didn't work out, anything else.

## Testing Instructions

* How to test this PR
Sign in with your account
Click on your profile, then 'Consent to Permissions'
Select any permissions. Notice that the groups do not collapse when permissions are selected
